docs/locales/docs.pot

```diff
@@ -1924,11 +1924,19 @@ msgid "Sort JSON keys"
 msgstr ""
 
 #: ../../admin/addons.rst:536
+msgid "``indent_type``"
+msgstr ""
+
+#: ../../admin/addons.rst:536
+msgid "JSON indentation type"
+msgstr ""
+
+#: ../../admin/addons.rst:538
 #: ../../admin/addons.rst:620
 msgid "``indent``"
 msgstr ""
 
-#: ../../admin/addons.rst:536
+#: ../../admin/addons.rst:538
```

Comment:
Changes to the POT files are not necessary, these are updated together with PO files.